workspace/__lib__/xod/math/is-finite/patch.xodp

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Determines if the given number has finite value i.e. it is normal, subnormal or zero, but not infinite or NaN",
```

Comment:
:scroll: Determines if the given number has a finite value, i.e., it is normal, subnormal, or zero, but not infinite or NaN